event-app-ui/src/components/Profile/Profile.jsx

```diff
@@ -57,6 +59,20 @@ export default function Profile() {
     });
   }, []);
 
+  useEffect(() => {
+    const currentDate = new Date();
+    const filterFutureEvent = eventData.filter((event) => {
+      const eventStartTime = new Date(event.starttime);
+      return eventStartTime > currentDate;
+    });
+    const filterPastEvent = eventData.filter((event) => {
+      const eventEndTime = new Date(event.endtime);
+      return eventEndTime < currentDate;
+    });
+    setFutureEvent(filterFutureEvent);
+    setPastEvent(filterPastEvent);
```

Comment:
Let's say an event is ongoing, where the start time is less than now and end time is greater than now. Would event show up here?

Reply:
I'll create a section for ongoing events in the next pull request